commands/events.py

```diff
@@ -0,0 +1,518 @@
+import json
+import os.path
+import time
+from datetime import datetime
+
+from disco.api.http import APIException
+from disco.bot import Plugin
+from disco.util import sanitize
+
+from commands.config import EventsPluginConfig
+from util import TrelloUtils
+
+
+@Plugin.with_config(EventsPluginConfig)
+class Events(Plugin):
+
+    def __init__(self, bot, config):
+        super().__init__(bot, config)
+        #initialize
+        self.queued = False
+        self.saving = False
+        self.status = "Scheduled"
+        self.participants = dict()
+        self.reported_cards = dict()
+
+    def load(self, ctx):
+        super(Events, self).load(ctx)
+        self.load_event_stats()
+
+    def unload(self, ctx):
+        self.save_event_stats()
+        super(Events, self).unload(ctx)
+
+    @Plugin.command("submit", "[submission:str...]")
+    def template(self, event, submission:str=None):
+        """Make a new submission"""
+        if self.status != "Started":
+            return #no event going on, pretend nothing happened #noleeks
+        if event.guild is None or event.channel.id != self.config.event_channel: #ignore users running this in the wrong channel, also prevents non hunters from submitting
+            return
+
+        help_message = "<@{}> It seems you're missing parts, the syntax for this command is `+submit <trello link> | <where this ticket should be moved to> | <why it should be moved there and/or new steps>`".format(event.author.id)
+        if submission is None:
+            #no params given, print help info
+            event.msg.reply(help_message)
+            return
+
+        parts = submission.split("|")
+        if len(parts) < 3:
+            #missing things we need
+            event.msg.reply(help_message)
+            return
+        if len(parts) > 3:
```

Comment:
after looking at this again, this is correct after all and no need for <=, if it's less then we need to error and exit out of handling as we are missing things

if we have more then 3 we need to re-assemble the final one as that got split up into more parts

but if it's exactly 3 we don't need to do any more processing and we have exactly how much we need (link, destination and info), if this was >= then it would add a | to it that isn't needed